### PR TITLE
Add the x64 and arm64 instructions for deb ES-oss install

### PR DESCRIPTION
**Pull request by sreekarjami**
*Issue #, if available:*

*Description of changes:*
Add the x64 and arm64 instructions for deb ES-oss install

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/install/deb.md [hunk 1]
```diff
@@ -42,11 +42,20 @@ These steps assume you're using Ubuntu 18.04.
...
    wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-{{site.es_version}}-amd64.deb
    sudo dpkg -i elasticsearch-oss-{{site.es_version}}-amd64.deb
    ```
 
+   Install ARM64
+
+   ```bash
+   wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-{{site.es_version}}-arm64.deb
+   sudo dpkg -i elasticsearch-oss-{{site.es_version}}-arm64.deb
+
```
Comment (aetter) [inline docs/install/deb.md:57]: Missing the closing backticks here. For general formatting, I'd prefer a single code block with both options, similar to how we present the Java 11 vs. Java 8 step here: https://opendistro.github.io/for-elasticsearch-docs/docs/install/rpm/

```
# x86
wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-{{site.es_version}}-amd64.deb
sudo dpkg -i elasticsearch-oss-{{site.es_version}}-amd64.deb
# ARM64
wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-{{site.es_version}}-arm64.deb
sudo dpkg -i elasticsearch-oss-{{site.es_version}}-arm64.deb
```

File: docs/install/rpm.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ Installing and running Open Distro for Elasticsearch from an RPM package is a mo
 
 1. Choose the version you'd like and install it:
 
+   The below command will pick the appropriate ARCH(x64 or ARM64) based on the system being used.
```
Comment (gaiksaya) [inline docs/install/rpm.md:43]: Maybe replace "ARCH" with architecture just to be clear. Avoid acronyms

File: docs/install/tar.md [hunk 3]
```diff
@@ -13,22 +13,38 @@ The tarball supports CentOS 7, Amazon Linux 2, Ubuntu 18.04, and most other Linu
 
 1. Download the tarball:
 
+   x64:
    ```bash
-   curl https://d3g5vo6xdbdb9a.cloudfront.net/tarball/opendistro-elasticsearch/opendistroforelasticsearch-{{site.odfe_version}}.tar.gz -o opendistroforelasticsearch-{{site.odfe_version}}.tar.gz
+   curl https://d3g5vo6xdbdb9a.cloudfront.net/tarball/opendistro-elasticsearch/opendistroforelasticsearch-{{site.odfe_version}}-x64.tar.gz -o opendistroforelasticsearch-{{site.odfe_version}}-x64.tar.gz
```
Comment (peterzhuamazon) [inline docs/install/tar.md:18]: All these tarball you miss the `linux` platform in the link.

Comment (sreekarjami) [reply docs/install/tar.md]: Made the change.

File: docs/install/rpm.md (hunk 2 above)
Comment (sreekarjami) [reply docs/install/rpm.md]: made the change.

File: docs/install/deb.md (hunk 1 above)
Comment (sreekarjami) [reply docs/install/deb.md]: I have made the change accordingly.

**Pull request merged**